docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+    * Domein/objectregistraties (domein/objectgegevens; meerdere registraties: één per domein/proces)
+    * Meldingenregistratie (ontvangen aanvragen en meldingen)
+    * Basisregistraties (personen, bedrijven, panden, terreinen, percelen e.d.)
+    * Evt. andere kernregistraties (Zaakobjecten die nog niet genoemd zijn, maar wel behoren tot kerngegevens (meervoudig gebruikt))
+
+
+
+Figuur 2: ZGW in een gegevenslandschap
+
+## 1.4 Vraagstukken
+
+Bovenstaande plaat schetst het concept ZGW in een gegevenslandschap al heel aardig.
+
+Toch zijn er nog enkele openstaande vragen te beantwoorden. Voor een deel is dit al af te leiden uit de stippellijnen in bovenstaande plaat:
+
+### 1.4.1. Hoe verhoudt melding/verzoek zich tot een zaak?
+
+In bovenstaande figuur is de relatie tussen Melding/Aanvraag (Verzoek) en Zaak als een stippellijn getekend. Dit omdat Verzoek niet in het RGBZ voorkomt (zat wel in GFO Zaken) en we dus niet weten hoe Verzoek en Zaak zich tot elkaar verhouden. Essentie van de relatie lijkt te zijn dat een verzoek of melding een op zich zelf staand ‘iets’ is en behandeld wordt als zaak. Één verzoek kan leiden tot meerdere zaken. Meerdere Verzoeken kunnen ook leiden tot één Zaak?
+Documenten bij een Verzoek worden bij het Verzoek vastgelegd. Later ook rechtstreeks gerelateerd aan de zaak?
+
+### 1.4.2. Waar slaan we procesinformatie op?
+
+Het is nog onduidelijk waar we in een gegevenslandschap “Procesinformatie” willen opslaan. In bovenstande figuur is dit – samen met procesdefinitiegegevens - gepositioneerd in de Zaakafhandelcomponent (net als nu in de diverse Taaksystemen).
+Dit is echter (mogelijk) in strijd met het uitgangspunt in Common ground dat functionaliteit en gegevens van elkaar gescheiden moeten zijn. Procesgegevens zijn immers ook gegevens.
+De procesdefinitie is in veel gevallen zelfs impliciet (hard gecodeeerde flow tussen schermen); dat valt dan niet eens los te knippen.
+
+Conclusie: hier is sprake van een glijdende schaal. Je wilt liefst alle domeingegevens en zo veel mogelijk procesgegevens uit de applicaties halen en via een gestandaardiseerde APi ontsluiten. We moeten echter wel oppassen dat we hier te ver in willen gaan en het paard daarmee achter de wagen spannen. In de praktijk zal dit voor veel bestaande taaksystemen een groeipad worden..
+
+Er is één maar. Het delen van procesinformatie is nodig om het handelen als overheid te verantwoorden: het moet dus gearchiveerd worden. Wie heeft op welke datum welke stap gezet? Welke procesflow (en keuzes) is er doorlopen? Welke informatie was wanneer beschikbaar? Hoe luidde de (deel)berekening die heeft geleid tot een bepaald resultaat? Dit is informatie die niet in één van de andere registraties past (tenzij als logfile in PDF in de DRC) en voor het grootste deel niet in de ZRC wordt vastgelegd (die kent meestal maar één behandelaar en vooral enkel het proces op hoofdlijnen: de Statussen).
```

Comment:
Wat is hier mis mee? Het procesverloop is toch puur bedoeld als verslaglegging?

Hoe archiveren we trouwens uiteindelijk de gerelateerde gegevens uit de andere registraties? Wat was de naam van de behandelaar, waar woonde de aanvrager op het moment van de aanvraag. Er zal toch op het moment van het afsluiten van de zaak een afslag van alle betrokken gegevens gemaakt moeten worden?

Of is hier een ander component voor? Of is dit ook proces informatie?

Reply:
tot nu toe is de enige gestructureerde manier die we hebben kunnen verzinnen en breed genoeg is om voor alle processen te kunnen werken het zaakgericht werken. En 'Zaak' is niet rijk genoeg om alle procesgegevens op te slaan die je eigenlijk zou moeten opslaan..

Reply:
>Hoe archiveren we trouwens uiteindelijk de gerelateerde gegevens uit de andere registraties? Wat was de naam van de behandelaar, waar woonde de aanvrager op het moment van de aanvraag. Er zal toch op het moment van het afsluiten van de zaak een afslag van alle betrokken gegevens gemaakt moeten worden?

>Of is hier een ander component voor? Of is dit ook proces informatie?

Goede vraag. Het antwoord moeten we zoeken in de richting van 'archiveren by design'. Het archief is geen aprt doosje meer, maar direct daar waar het is opgeslagen moet het duurzaam bewaard worden..